app/authentication/flush_permission_denied.py

```diff
@@ -0,0 +1,8 @@
+
+class FlushPermissionDenied(Exception):
```

Comment:
This should probably be in a file/class appended with 'Exception/_exception'

Also seems to be an empty line at the top here.